### PR TITLE
Check in heroku setup, deploy and restart scripts

### PR DESCRIPTION
**Pull request by grodowski**
@jbarber that's the change we discussed today - scripts included in the image will be available in the build environment, so that we don't have to copy & paste them in future circle 2.0 migrations

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,8 @@
 # heroku-deploy
 
 A Dockerfile that we use to deploy stuff to Heroku with.
+
+### Configuration
+
+1. Expose heroku credentials to the container via: `HEROKU_LOGIN` and `HEROKU_API_KEY` variables (set in CircleCI's UI or `config.yml`)
+2. Execute `setup-heroku.sh` that stores the credentials in `~/.netrc` before calling any script.
```
Comment (jbarber) [inline README.md]: Change the scripts so they call `setup-heroku.sh` first? Then we can lose this instruction.

**Review by jbarber — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (jbarber) [reply README.md]: It also means we don't need to do this step in all the circle.yml's?

Comment (jbarber) [reply README.md]: It also means we don't need to do it in all the circle.yml's?

Comment (grodowski) [reply README.md]: Sounds good, I'll change it!

**Review by jbarber — APPROVED** [review]

**Pull request merged**